bulk_writer.js

```diff
@@ -140,7 +140,11 @@ BulkWriter.prototype.checkEsConnection = function checkEsConnection() {
   return new Promise((fulfill, reject) => {
     operation.attempt((currentAttempt) => {
       debug('checking for connection');
-      thiz.client.ping().then(
+      thiz.client.cluster.health({
+        timeout: '5s',
+        wait_for_nodes: '1',
+        wait_for_status: 'yellow'})
```

Comment:
yellow instead of green, because of a single node Elasticsearch with default "number_of_replicas" 1 for indices can get green status at all.